release/assets/macos_cli_setup.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash -e
+
+USER_PROVIDED_PATH=$1
+
+print_help() {
+  cat << EOF
+
+Conjur CLI setup script
+
+Installs and configures the Conjur CLI on macOS environments.
+Upon completion of this setup script, you will be able to access the Conjur CLI from anywhere on your machine.
+The installation creates a symbolic link between the CLI executable and directory path. By default, this path is '/usr/local/bin'.
+To overwrite this path, configure the directory path parameter. For example '$0 /usr/bin'
+
+Usage: $0 [options]
+    -h, --help    Shows this help message
+
+EOF
+  exit
+}
+
+main() {
+  cat << EOF
+   ______
+  / ____/___  ____    ( )_  ________
+ / /   / __ \/ __ \  / / / / // ___/
+/ /___/ /_/ / / / / / / /_/ // /
+\____/\____/_/ /_/_/ /\____//_/
+                /___/
+EOF
+
+  if [[ $USER_PROVIDED_PATH = "--help" ]]; then
+    print_help
+  fi
+  executable_path=${USER_PROVIDED_PATH:-'/usr/local/bin'}
+
+  if [[ ! -d "/usr/local/bin" ]]; then
+    echo "The default directory '/usr/local/bin' does not exist. Provide the path to the user directory to create the symbolic link. For example: '$0 /usr/bin'"
```

Comment:
Not sure about ux of this message . Maybe it better to give specific example and not with $0?

Reply:
$0 will turn into the path that the user ran the script. For example if I'm in ./release/assets/conjur then that will take the $0 value